doc-surrealdb_versioned_docs/version-1.x/tutorials/define-a-schema.mdx

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 6
+sidebar_label: Define a Schema in SurrealDB
+title: Define a Schema in SurrealDB | Tutorials
+description: In this tutorial, you will learn how to define a schema in SurrealDB, what using either schema type means for data retrieval, and how to use what you need as your product grows.
+---
+
+# Define a Schema in SurrealDB
+
+
+When starting a new database project, there are a couple of early decisions to be made, such as creating tables and defining the records that will be in these tables and also the datatypes of the records, and how the tables you describe relate to each other which includes data sharing or manipulation.
+
+While this might look like many decisions, you are Defining a Schema.
```

Comment:
Talking up the benefits of having a schema here, I assume? Maybe a change like this will work:

"you are Defining a Schema" -> "defining a schema is the primary way to ensure that your data behaves in the way you expect it to."